### ISSUE TITLE
There is no use of val_dataset in train.py?

### ISSUE THREAD

**Issue by whuzt** [Tianxiaomo/pytorch-YOLOv4#80]

**Comment by Tianxiaomo** [comment]
Yes, I originally planned to write verification, but I have been busy recently.

**Comment by wenh06** [comment]
> Yes, I originally planned to write verification, but I have been busy recently.

Please review the [pull request](https://github.com/Tianxiaomo/pytorch-YOLOv4/pull/136)

**Comment by Tianxiaomo** [comment]
> > Yes, I originally planned to write verification, but I have been busy recently.
> 
> Please review the [pull request](https://github.com/Tianxiaomo/pytorch-YOLOv4/pull/136)

Thank you very much for adding verification
